src/components/CircularProgress/CircularProgress.tsx

```diff
@@ -0,0 +1,73 @@
+import React, { useId } from 'react';
+import { tokens } from '@/DesignTokens';
+
+import classes from './CircularProgress.module.css';
+
+export interface CircularProgressProps {
+  value: number;
+  width?: number;
+  strokeWidth?: number;
+  ariaLabel?: string;
+  label?: string;
+  id?: string;
+}
+
+export const CircularProgress = ({
+  value,
+  width = 70,
+  strokeWidth = 2.5,
+  ariaLabel,
+  label,
+  id = 'progress',
```

Comment:
Should we perhaps make the `id` required instead? If you leave it out, and have multiple progressbars (although I guess thats an edge case as well), we will have multiple identical ID's which is not valid, and I guess will be confusing for screen readers

Reply:
Agree